spec/amp-var-substitutions.md

```diff
@@ -683,6 +685,12 @@ requested attribute names and the object values are the elements' values for tho
 * The CSS selector may contain only letters (upper- and/or lower-case), numbers, hyphens, underscores, and periods.
 [Issue #14252](https://github.com/ampproject/amphtml/issues/14252) has been created to address potential future demand for more complex CSS selectors.
 
+#### Initial Scroll Depth
+
+Provides the scroll depth at the time the element was loaded, relative to the target. The value will indicate whether the scroll position was above/at/below the target.
```

Comment:
nit: `inside/top/bottom`. Please indicate the resolved value is a string of the three.

Reply:
Thanks! Done.